src/add-field-dropdown/add-field-dropdown.component.html

```diff
@@ -1,8 +1,8 @@
 <div *ngIf="!disabled" class="btn-group" dropdown keyboardNav="true" (onToggle)="focusElementIfOpen($event, expressionInput)">
-  <button id="simple-btn-keyboard-nav" type="button" class="editor-btn-add-subfield" dropdownToggle>+ <span class="caret"></span></button>
+  <button id="simple-btn-keyboard-nav" type="button" class="btn-add-field-dropdown" dropdownToggle><ng-content></ng-content> <span class="caret"></span></button>
```

Comment:
`<ng-content></ng-content>` is to make `add-field-dropdown` title dynamic, so that we can set it where we define the dropdown like below.

```
<add-field-dropdown...>Add a field</add-field-dropdown>
```

The same thing can be done via `@Input` but I felt like this is more natural.